src/Attachment.js

```diff
@@ -0,0 +1,176 @@
+/*
+    Copyright 2014 Rustici Software
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+/**
+TinCan client library
+
+@module TinCan
+@submodule TinCan.Attachment
+**/
+(function () {
+    "use strict";
+
+    /**
+    @class TinCan.Attachment
+    @constructor
+    */
+    var Attachment = TinCan.Attachment = function (cfg) {
+        this.log("constructor");
+
+        /**
+        @property usageType
+        @type String
+        */
+        this.usageType = null;
+
+        /**
+        @property display
+        @type Object
+        */
+        this.display = null;
+
+        /**
+        @property contentType
+        @type String
+        */
+        this.contentType = null;
+
+        /**
+        @property length
+        @type int
+        */
+        this.length = null;
+
+        /**
+        @property sha2
+        @type String
+        */
+        this.sha2 = null;
+
+        /**
+        @property description
+        @type Object
+        */
+        this.description = null;
+
```

Comment:
Should include `content` in the prototype since it will sometimes be set, then can just check for `null` to see if it *has* been set.